Xamarin.Forms.Platform.Android/FastRenderers/ButtonRenderer.cs

```diff
@@ -241,12 +241,6 @@ protected override void OnLayout(bool changed, int l, int t, int r, int b)
 			base.OnLayout(changed, l, t, r, b);
 		}
 
-		protected override void OnMeasure(int widthMeasureSpec, int heightMeasureSpec)
-		{
-			_buttonLayoutManager?.Update();
```

Comment:
@mattleibow if you can recall the reason for adding this code let me know please. I tested a few different scenarios with loading images through urls and loading them after the layout was set to see if removing this broke anything but I couldn't find a scenario where deleting this caused a problem